scripts/uberenv/packages/conduit/package.py

```diff
@@ -0,0 +1,498 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
```

Comment:
I don't think this is the latest conduit spack pkg -- it should have version 0.5.1 in it)

Reply:
See my comment below

Reply:
My mistake, I added a configure phase in ascent, conduit already at it.

Reply:
Update will be straightforward.